src/utils/error.js

```diff
@@ -6,6 +6,9 @@ function handleError(rawError) {
   let error;
   // NOTICE: We check if the errors are from the API (have a status) or if thrown internally.
   if (rawError.status) {
+    if (rawError.status === 500) {
+      return UNEXPECTED_ERROR_MESSAGE;
+    }
```

Comment:
I took the opportunity to be sure we'll never have this kind of message:
![image](https://user-images.githubusercontent.com/48474636/82462409-d4ebb380-9abb-11ea-9e78-937b3c55cf45.png)

Reply:
[MISC] I'm not quite sure that we should do something like that. 500 errors shouldn't exist, that mens we should look for it directly when needed.

IMO we should have those displayed